test/DynamoCoreWpf3Tests/UnitsUITests.cs

```diff
@@ -198,7 +198,6 @@ public void ForgeUnitDropdownsLoadWithMalformedData()
             Assert.AreNotEqual("autodesk.unit.quantity:force", GetTypeName<DynamoUnits.Quantity>(node3.CachedValue.Data));
             Assert.AreEqual("autodesk.unit.quantity:flowPerVolume", GetTypeName<DynamoUnits.Quantity>(node3.CachedValue.Data));
             Assert.AreEqual("autodesk.unit.symbol:mm", GetTypeName<DynamoUnits.Symbol>(node4.CachedValue.Data));
-            Assert.AreEqual("autodesk.unit.unit:maxwells", GetTypeName<DynamoUnits.Unit>(node5.CachedValue.Data));
```

Comment:
The `maxwells` schema was selected because the dropdown node used index `221` in the malformed DYN node JSON data (since _"Millimeters AAA"_ is not a valid unit schema name). This index maps to different units (`maxwells` or `microarcseconds`) depending on whether ASC schemas are present. This test condition was removed as it doesn't provide meaningful validation of the dropdown selection system's stability, which is already functioning correctly.